backend/expense/src/main/java/org/egov/digit/expense/web/models/IndividualSearch.java

```diff
@@ -0,0 +1,30 @@
+package org.egov.digit.expense.web.models;
```

Comment:
@ajay-egov we can not copy IndividualSearch model here. use Map to create these.